mungegithub/submit-queue/deployment/kubernetes/configmap.yaml

```diff
@@ -16,14 +16,14 @@ github-key-file: /etc/hook-secret-volume/secret
 # status context options.
 required-contexts: ""
 required-retest-contexts: "\
```

Comment:
The list doesn't need to be sorted though. I think a comment that indicates why ordering is important without enforcing an unnecessary constraint would be better.  How about `# If multiple contexts in these lists are failing, the first failure in the list is used as the reason for not merging.`?